src/edu/stanford/dlss/wowza/SulWowza.java

```diff
@@ -121,9 +147,61 @@ public void play(IClient client, RequestFunction function, AMFDataList params)
         }
 	}
 
+    /**
+     * Initalizes the Honeybadger error reporting tool. This is a public method so we can call
+     * it from the tests. It's outside the constructor, since testing constructors with Mockito is a pain.
+     */
+    public void initHoneybadger()
+    {
+        if(environment == null)
+            environment = new SulEnvironment();
+
+        String apiKey = environment.getEnvironmentVariable(HONEYBADGER_KEY);
+        if(apiKey == null)
+        {
+            getLogger().error(this.getClass().getSimpleName() + " unable to set up Honeybadger error reporting (missing API key environment variable?)");
+            invalidConfiguration = true;
+        }
+        else
+        {
+            honeybadgerConfig = new StandardConfigContext();
+            honeybadgerConfig.setApiKey(apiKey)
+                             .setEnvironment(HONEYBADGER_ENVIRONMENT)
+                             .setApplicationPackage(this.getClass().getPackage().getName());
+        }
+    }
+
 
     // --------------------------------- the public API is above this line ----------------------------------------
 
+    void registerUncaughtExceptionHandler()
```

Comment:
I have a slight preference to put the stuff we're less likely to be tracing through lower down, so I would probably have added these methods at the bottom, maybe even with a block comment that these are the exception handling methods for HoneyBadger.

This is definitely a quibble, so if you disagree, then leave the code here and I'm ok with it.